### PR TITLE
Implemented MediumTicketSeller::assignSeat

### PR DESCRIPTION
**Pull request by FMock**
Implemented.

### CODE REVIEW

File: MediumTicketSeller.cpp [hunk 1]
```diff
@@ -17,9 +17,35 @@ int MediumTicketSeller::sellTime() {
...
+		}
+		else
+		{
+				row += 20%c;
+		}
+		c--;
+	}
 	
 	// if an open seat is found, place the customer there (customerName)
 	if (true) {
```
Comment (flip40) [inline MediumTicketSeller.cpp]: this is missing the output check, if (seatFound)...

File: MediumTicketSeller.cpp [hunk 2]
```diff
@@ -17,9 +17,35 @@ int MediumTicketSeller::sellTime() {
 }
 
 bool MediumTicketSeller::assignSeat(string customerName) {
-	
+	bool seatFound = false;
 	// search seats for an open seat according to the priority
+	int c = 19;
 	
+	for(int row = 4; 20%c > 0;)
```
Comment (flip40) [inline MediumTicketSeller.cpp]: Won't this go 10 rows in each direction?

**Comment by flip40** [comment]
I believe this misses row 9.

**Comment by FMock** [comment]
oops!
Fixed now

**Comment by flip40** [comment]
The loop also appears to miss the last row (row 10, index 9). I tested it in a separate file, so perhaps I implemented it wrong, but this is what I am seeing.
Code:
```
	int c = 19;
	
	for (int row = 4; 20 % c > 0;) {
		std::cout << "row " << row + 1 << "\n";
		
		// inner for loop for columns here
		
		if (20 % c % 2 == 0) row -= 20 % c;
		else row += 20 % c;
		c--;
	}
```
Output:
```
row 5
row 6
row 4
row 7
row 3
row 8
row 2
row 9
row 1
```

**Comment by FMock** [comment]
I just ran it in a separate program myself. Loop works good but just stops short of the last row (row 9)
Remember row 9 is actually row 10
[output.txt](https://github.com/CS149-3/homework3/files/149710/output.txt)

**Comment by FMock** [comment]
Okay I figured out what was wrong. Take a  look. now all rows and cols are gone through
I'll update my branch to reflect this very soon
[fixed.txt](https://github.com/CS149-3/homework3/files/149724/fixed.txt)

**Comment by FMock** [comment]
MediumTicketSeller-assignSeat works correctly now

**Comment by flip40** [comment]
Awesome, merging.

**Pull request merged**